--- install.sh	2021-02-05 09:16:34.000000000 -0500
+++ install.sh	2021-02-06 19:23:01.000000000 -0500
@@ -19,7 +19,7 @@
 
 #bootstrap by checking that the MakeMaker module is installed
 module="ExtUtils::MakeMaker"
-perl -M$module -e 1 2> /dev/null
+__PERL__ -M$module -e 1 2> /dev/null
 status=$?
 if ( test $status -ne 0 ) then
   echo "Cannot locate the $module Perl module"
@@ -115,7 +115,7 @@
 
 #install user-interface scripts
 #perl Makefile.PL INSTALL_BASE=$TOP
-perl Makefile.PL INSTALL_BASE=$TOP INSTALLSITESCRIPT=$TOP/bin INSTALLSITEBIN=$TOP/bin INSTALLSITELIB=$TOP/lib INSTALLSITEMAN1DIR=$TOP/share/man/man1 INSTALLSITEMAN3DIR=$TOP/share/man/man3
+__PERL__ Makefile.PL INSTALL_BASE=$TOP INSTALLSITESCRIPT=$TOP/bin INSTALLSITEBIN=$TOP/bin INSTALLSITELIB=$TOP/lib INSTALLSITEMAN1DIR=$TOP/share/man/man1 INSTALLSITEMAN3DIR=$TOP/share/man/man3
 stat=$?
 if ( test $stat -ne 0 ) then
   exit $stat
