--- install.sh	2019-05-21 12:42:38.000000000 -0400
+++ install.sh	2019-06-08 20:02:51.000000000 -0400
@@ -19,7 +19,7 @@
 
 #bootstrap by checking that the MakeMaker module is installed
 module="ExtUtils::MakeMaker"
-perl -M$module -e 1 2> /dev/null
+__PERL__ -M$module -e 1 2> /dev/null
 status=$?
 if ( test $status -ne 0 ) then
   echo "Cannot locate the $module Perl module"
@@ -102,7 +102,7 @@
 
 #install user-interface scripts
 #perl Makefile.PL INSTALL_BASE=$TOP
-perl Makefile.PL INSTALLSITESCRIPT=$TOP/bin INSTALLSITEBIN=$TOP/bin INSTALLSITELIB=$TOP/lib INSTALLSITEMAN1DIR=$TOP/share/man/man1 INSTALLSITEMAN3DIR=$TOP/share/man/man3
+__PERL__ Makefile.PL INSTALLSITESCRIPT=$TOP/bin INSTALLSITEBIN=$TOP/bin INSTALLSITELIB=$TOP/lib INSTALLSITEMAN1DIR=$TOP/share/man/man1 INSTALLSITEMAN3DIR=$TOP/share/man/man3
 stat=$?
 if ( test $stat -ne 0 ) then
   exit $stat
